pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/controller/LoginController.java

```
@@ -92,11 +93,11 @@ public void initBinder(WebDataBinder binder) {
 	 * @return AuthNResponse
 	 */
 	@PostMapping(value = "/sendOtp", produces = MediaType.APPLICATION_JSON_VALUE)
```

deprecate this. Have a flag (mosip.preregistration.sendotp.allowapi=false) to disable this API. We should throw an exception with a documented error code based on this flag (flag set to false). We should ensure to have this flag as false and ask UI team to quickly move to the new api.

---

have added a flag as mentioned